### ISSUE TITLE
Latency Placing Mekanism Conduits

### ISSUE THREAD

**Issue by Utildayael** [amadornes/MCMultiPart#52]
I'm told by the Mekanism dev @aidanbrady that this is an issue with MultiPart?

Here's the Mek issue I logged:

https://github.com/aidancbrady/Mekanism/issues/3751

The problem is when any of his pipes are placed OR removed there is major latency between the time you click to place and the time it actually places. Super laggy.

Can you please look at the linked issue and address? If not a Multipart issue, any help in "where to take the issue" would be appreciated.

MC 1.10.2
MCMultiPart 1.2.1
Mekanism 9.1.1
Forge 2065

Having similar issue with EnderIO but haven't got confirmation from that dev that he uses multipart for anything. Will update when I hear back on that one.

Thanks :)

**Comment by amadornes** [comment]
This sounds like just a server-client lag issue.
Vanilla breaks blocks on the client and then sends an update to the server. If the sever is okay with it, it's broken there as well and synced to other players. If it's not, it sends an update to the player that broke the block putting it back where it was.
In MCMultiPart and EnderIO, the client doesn't do the breaking, it happens on the server. Once the client is done breaking a part, it's not removed until the server says so.
This is most likely what's causing the breaking issues and there isn't all that much I can do about it. I'm working on redesigning MCMP's internals, which could possibly "get around this"... but I can't say it will for sure. What I can say almost for sure is that the delay is caused by lag.

**Comment by Utildayael** [comment]
Hrm interesting. I'll talk with my server admin and see if we can profile things. Only seems to be with EnderIO and Mek pipes oddly. 

Normal blocks, even Chisel & Bits blocks, break as expected. 

Appreciate the feedback. :)

**Comment by amadornes** [comment]
Any news? I still haven't been able to reproduce it :/

**Comment by Utildayael** [comment]
Apologies. My server admin moved a few months ago and is still trying to get internet access back (!) so he can't update the pack. I can't repro the problem on a SP game sadly so I'm kind of waiting on him. He moved to a new area and the builder forgot (!) to tell the local cable provider to say... install fiber lines and stuff. Poor guy hasn't had internet in literally months. 

I'll keep ya posted or you can close this out for now until He can update and repro. 

Thanks!

**Comment by amadornes** [comment]
Ouch... No rush, though :)

**Comment by dblsaiko** [comment]
Closing because this is an MCMP1 issue which has been outdated for a while. If this issue still occurs in MCMP2 (1.11+), please open a new issue.
